### ISSUE TITLE
The intercom box is on top of the charter (cuts the text and button)

### ISSUE THREAD

**Issue by dberesford** [CoderDojo/community-platform#645]
![screen shot 2015-09-16 at 09 36 41](https://cloud.githubusercontent.com/assets/602445/9900131/8cd9ab72-5c56-11e5-8538-c815f0596ed5.png)

**Comment by tangentfairy** [comment]
__Technical spec__

Charter template is here https://github.com/CoderDojo/cp-zen-platform/blob/cf2e0c2eaecd208b91fc1bba1e2871e8f155059b/web/public/templates/charter/index.dust 

CSS file is here https://github.com/CoderDojo/cp-zen-platform/blob/cf2e0c2eaecd208b91fc1bba1e2871e8f155059b/web/public/css/dashboard.less

**Comment by flakyfilibuster** [comment]
hey all, I had a look at this. 
I'm not quite sure this is solvable in the current state. If you see the intercom as being a problem on the charter page - then it's a problem during the whole registration. It will always overlay something.

Also, what screen dimensions are we talking here? 
One could argue, that it is no problem from 1800px upwards, on the other hand the intercom overlays the whole screen on mobile view - where is the cutoff, when do we say: 'this is fixed'.

Long story short, the fundamental design is behaving as expected. 
Changing it only for the charter page won't fix the behavior of the general design.

Do let me know if I'm totally off base here :)

**Comment by tangentfairy** [comment]
@flakyfilibuster I think it'd be a good solution to at least align the button to meet with the right hand side of the signature input. I'd be happy with that anyway :)

**Comment by flakyfilibuster** [comment]
hey there, sorry it took so long.
This got fixed in: https://github.com/CoderDojo/cp-zen-platform/pull/815

Please see attached screenshots.
![screen shot 2016-02-08 at 7 53 05 am](https://cloud.githubusercontent.com/assets/2176610/12879849/cde2754a-ce39-11e5-860b-f88838e70ec9.png)

**Comment by tangentfairy** [comment]
This is live! Thanks @flakyfilibuster
